Add `Community Mentorship Program Summer 2023: Signups Now Open` news post (#9470)
ppy/osu-wiki: news/2023/2023-06-05-community-mentorship-program-summer-2023-signups-now-open.md

---
layout: post

> title: "Community Mentorship Program Summer 2023 Signups Now Open"

Reviewer: I think this looks better
```suggestion
title: "Community Mentorship Program Summer 2023: Signups Now Open"
```

date: 2023-06-05 10:00:00 +0000
---

The 24th cycle of the Community Mentorship program is on its way! Get ready to delve deep into the world of mapping, modding, or storyboarding.

![](/wiki/shared/news/banners/community-mentorship-program.jpg)

The Community Mentorship program serves as a place for upcoming mappers, modders, storyboarders, and even prospective mentors to develop their skills under the guidance of experienced members of the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.

Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events (like contests), hang out with fellow mentees, and more!

> If you're experienced in mapping, modding, or storyboarding, and have a desire to take those who are new under your wing, then consider joining us as a mentor! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1716981), you'll be able to apply right away. If you're not sure if you'll make the cut, we encourage you apply anyways! You could be offered to participate in the Graduation Program as a Junior Mentor, where our most experienced mentors will teach you how to effectively mentor.

Reviewer: this is subjective
```suggestion
If you're experienced in mapping, modding, or storyboarding, and have a desire to take those who are new under your wing, then consider joining us as a mentor! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1716981), you'll be able to apply right away. If you're not sure if you'll make the cut, we encourage you to apply anyways! You could be offered to participate in the Graduation Program as a Junior Mentor, where our most experienced mentors will teach you how to effectively mentor.
```

> If you're experienced in mapping, modding, or storyboarding, and have a desire to take those who are new under your wing, then consider joining us as a mentor! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1772532), you'll be able to apply right away. If you're not sure if you'll make the cut, we encourage you to apply anyways! You could be offered to participate in the Graduation Program as a Junior Mentor, where our most experienced mentors will teach you how to effectively mentor.

Reviewer 2: ```suggestion
If you're experienced in mapping, modding, or storyboarding, and have a desire to take those who are new under your wing, then consider joining us as a mentor! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1772532), you'll be able to apply right away. If you're not sure whether you'll make the cut, we encourage you to apply anyway! You could be offered to participate in the Graduation Program as a Junior Mentor, where our most experienced mentors will teach you how to effectively mentor.
```
